content/en/docs/components/model-registry/getting-started.md

```diff
@@ -0,0 +1,169 @@
++++
+title = "Getting started"
+description = "Getting started with Model Registry using examples"
+weight = 30
+
++++
+
+This guide shows how to get started with Model Registry and run a few examples using the
+command line or Python clients.
+
+At this time, there is not yet an UI for Model Registry; therefore, this documentation focus on backend services and APIs.
+
+For an overview of the logical model of model registry, check the
+[Model Registry logical model](https://github.com/kubeflow/model-registry/blob/main/docs/logical_model.md).
+The logical model is exposed via the Model Registry [REST API](https://editor.swagger.io/?url=https://raw.githubusercontent.com/kubeflow/model-registry/main/api/openapi/model-registry.yaml).
+
+## Prerequisites
```

Comment:
Would be nice to add min Kubernetes and Python version to install MR, like here: https://www.kubeflow.org/docs/components/katib/installation/#prerequisites

Reply:
This is about tutorial to _use_ MR after installation for which prerequisite are same per Kubeflow, here: https://github.com/kubeflow/website/pull/3698/files#diff-c3b16833ded8b5282aa1d0c8f6caf09c31b1e44b47f27086757b195a9031f9e8R14-R15

But great catch, thanks! 
Added python version for SDK with: https://github.com/kubeflow/website/pull/3698/commits/9e4c03c1c274d31cf0d5762a719051d4871cf86c 🚀

Reply:
marking as resolved following 👍 on previous comment.